cmake/XSDKOptions.cmake

```diff
@@ -13,9 +13,9 @@ set(USE_XSDK_DEFAULTS OFF CACHE BOOL "Use XSDK defaults?")
 if(USE_XSDK_DEFAULTS)
   message(STATUS "XSDK defaults are enabled.")
   # 4. Debug and Release
-  set(CMAKE_BUILD_TYPE Debug CACHE STRING "The build type.")
+  set(CMAKE_BUILD_TYPE Debug CACHE STRING "The build type." FORCE)
```

Comment:
Why this `FORCE`?

Reply:
This forces to override variables already located in the CMake cache.